lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -85,6 +90,36 @@ export class PublicClientApplication extends ClientApplication implements IPubli
         if (!account) {
             throw BrowserAuthError.createNoAccountError();
         }
+        const thumbprint: RequestThumbprint = {
+            clientId: this.config.auth.clientId,
+            authority: request.authority || "",
+            scopes: request.scopes,
+            homeAccountIdentifier: account.homeAccountId
+        };
+        const silentRequestKey = ThrottlingUtils.generateThrottlingStorageKey(thumbprint);
+        let response = this.silentRequest.get(silentRequestKey);
+        if (typeof response === "undefined") {
```

Comment:
Can we add a logger statement here, similar to teh one below for the else?

Reply:
Done